src/toil/common.py

```diff
@@ -617,20 +790,21 @@ def addOptions(parser: ArgumentParser, config: Optional[Config] = None, jobstore
                     "as well as parameters to control the level of provisioning."
     )
     provisioner_choices = ['aws', 'gce', None]
+
     # TODO: Better consolidate this provisioner arg and the one in provisioners/__init__.py?
-    autoscaling_options.add_argument('--provisioner', '-p', dest="provisioner", choices=provisioner_choices,
+    autoscaling_options.add_argument('--provisioner', '-p', dest="provisioner", choices=provisioner_choices, default=None,
                                      help=f"The provisioner for cluster auto-scaling.  This is the main Toil "
                                           f"'--provisioner' option, and defaults to None for running on single "
                                           f"machine and non-auto-scaling batch systems.  The currently supported "
-                                          f"choices are {provisioner_choices}.  The default is {config.provisioner}.")
-    autoscaling_options.add_argument('--nodeTypes', default=None,
+                                          f"choices are {provisioner_choices}.  The default is {None}.")
+    autoscaling_options.add_argument('--node_types', default=[], dest="nodeTypes", type=parse_node_type, nargs="*",
                                      help="Specifies a list of comma-separated node types, each of which is "
                                           "composed of slash-separated instance types, and an optional spot "
                                           "bid set off by a colon, making the node type preemptible. Instance "
                                           "types may appear in multiple node types, and the same node type "
                                           "may appear as both preemptible and non-preemptible.\n"
                                           "Valid argument specifying two node types:\n"
-                                          "\tc5.4xlarge/c5a.4xlarge:0.42,t2.large\n"
+                                          "\t[c5.4xlarge/c5a.4xlarge:0.42, t2.large]\n"
```

Comment:
```suggestion
                                          "\tc5.4xlarge/c5a.4xlarge:0.42,t2.large\n"
```

I don't think we're actually able to parse brace-enclosed, space-separated lists form the command line are we?